ci/docker/runtime_functions.sh

```diff
@@ -343,6 +343,11 @@ build_ubuntu_cpu_openblas() {
         USE_BLAS=openblas             \
         USE_DIST_KVSTORE=1            \
         -j$(nproc)
+
+    export CC="gcc"
+    export CXX="g++"
+    make cython PYTHON=python2
```

Comment:
ccache doesn't work?

Reply:
ccache does not work. (http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/incubator-mxnet/detail/PR-10951/11/pipeline).